web/vibe/web/rest.d

```diff
@@ -1641,6 +1641,7 @@ private HTTPServerRequestDelegate jsonMethodHandler(alias Func, size_t ridx, T)(
 				auto ret = () @trusted {
 					return evaluateOutputModifiers!CFunc(
 						__traits(getMember, inst, Method)(params), req, res); } ();
+				res.headers["Content-Type"] = "application/octet-stream";
```

Comment:
Wouldn't this overwrite the `Content-Type` a user sets using `@headerParam` ?

Reply:
I guess, although I'm unsure why you would want to set the content type using a header parameter. `@contentType` doesn't appear to be supported by the REST code so far, which is what I thought should be minded here.

Reply:
It should be said, though, that the other paths using `writeBody` have already been overwriting the content type until now. IMO this is okay until `@contentType` support gets added.